platform/osx/export/export.cpp

```diff
@@ -41,6 +41,7 @@
 #include "platform/osx/logo.gen.h"
 #include "string.h"
 #include "version.h"
+#include <sys/stat.h>
```

Comment:
Just a style comment, please leave an empty line between Godot headers (`""` includes) and system headers (`<>` includes) for readability :)